finality-aleph/src/sync/forest/mod.rs

```diff
@@ -79,6 +79,35 @@ impl Display for Error {
     }
 }
 
+/// What can go wrong when creating the forest.
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub enum InitializationError<B, J, CS>
+where
+    B: Block,
+    J: Justification<Header = B::Header>,
+    CS: ChainStatus<B, J>,
+{
+    Error(Error),
+    ChainStatus(CS::Error),
+}
+
+impl<B, J, CS> Display for InitializationError<B, J, CS>
+where
+    B: Block,
+    J: Justification<Header = B::Header>,
+    CS: ChainStatus<B, J>,
+{
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result<(), FmtError> {
+        match self {
+            InitializationError::Error(e) => match e {
+                Error::TooNew => write!(f, "Error during initialization. There are more imported non-finalized blocks in the Substrate database that can fit into the Forest. Purge the block database and restart the node."),
+                e => write!(f, "Error during initialization: {}", e),
+            },
+            InitializationError::ChainStatus(e) => write!(f, "ChainStatus error during initialization: {}", e),
+        }
```

Comment:
The "during initialization" things here are repeating themselves with the wrapper in handler – in general when writing an error message you want it to contain only internal information, not about the expected context.

On a similar note – error messages should not end with `.`, as this will get repeated in the log message. It's also better if they don't start with capital letters.

So imo better versions of this messages are, in order:
1. "there are more imported non-finalized blocks in the database that can fit into the forest – purge the block database and restart the node"
2. "{}" (yes, just the underlying error)
3. "chain status error: {}"

Reply:
fixed